share/wake/lib/system/sources.wake

```diff
@@ -90,6 +91,7 @@ export def claim (file: String): Result Path Error =
             # Its important that the command here is distinct from what `source` would use
             # so that claim and source do not overlap.
             makeExecPlan ("<claim>", str time, file, Nil) Nil
+            | setPlanLabel "<claim> {file}"
```

Comment:
Now that we're separating the label from the command, do we want to change the format?  The `<name>` angle brackets were only required to ensure it wouldn't clash with some tool/script which also happened to be called `claim`.
```suggestion
            | setPlanLabel "claim: {file}"
```